css/example/Modal.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../components/Modal.css">
+    <script src="https://kit.fontawesome.com/dcbd873a92.js" crossorigin="anonymous"></script>
```

Comment:
To use font awesome scripts are not needed. Can be handled with link. Please look for font awesome cdn

Reply:
how to use it

Reply:
done